### DISCUSSION TITLE
Where is setting.json file on Windows?

### DISCUSSION THREAD

**Discussion by RRRainick** [localsend/localsend#1411] [Discussion (English, International)]
hello, may I ask where is setting.json(the file that contains localsend setting) on windows? I learned that localsend config is store in `XDG_DATA` from [previous discussion](https://github.com/localsend/localsend/discussions/594#discussioncomment-6579570), but it seems XDG standard is not suitable for windows.

**Comment by Tienisto** [comment]
I think in appdata / roaming. The folder is named "org.localsend".

**Reply by RRRainick** [reply]
I install localsend using winget, and didn't find any folder named "org.localsend" in %appdata%(`~\AppData\Roaming`).
but I do found a folder named "11157TienDoNam.LocalSend_g71aj367vb7ze" in %localappdata%(`~\AppData\Local`), which contains "settings.dat" file. however it is unreadable with any text editor, seems to be a binary file.

**Reply by Tienisto** [reply]
Is there the exe? Then you can create and empty settings.json file

**Reply by RRRainick** [reply]
no, there is no executable file in %localappdata%. there's the directory structure.
![image](https://github.com/localsend/localsend/assets/47151049/a6c18124-c908-4aba-87ed-27855bab92ca)
